be/src/Project/Services/Identity/Identity.Api/Areas/Accounts/Resources/AccountResources.en.resx

```diff
@@ -147,6 +147,9 @@
   <data name="ForgotPasswordLabel" xml:space="preserve">
     <value>Password reset</value>
   </data>
+  <data name="InvalidPassword" xml:space="preserve">
+    <value>Incorrect password. Please try again or click "Password reset" to reset it.</value>
```

Comment:
**suggestion:** please make the error message more obscure. It should be "Incorrect e-mail or password. Please try again or click "Password reset" to reset it."